feature-libs/product-configurator/rulebased/root/cpq/cpq-configurator-overview.module.ts

```diff
@@ -0,0 +1,73 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { CmsConfig, provideDefaultConfig } from '@spartacus/core';
+import { CmsPageGuard, PageLayoutComponent } from '@spartacus/storefront';
+
+/**
+ * Takes care of the configuration overview that visualizes the attribute value assignments that have been done already in a condensed, read-only form.
+ * The end-user can switch between the interactive view and this overview.
+ * Provides routing, assignment of ng components to CMS components and assignment of CMS components to the layout slots.
+ * Some of the ng components on this view (tab bar, price summary and addToCart button) are shared between the interactive view and the overview.
+ */
+@NgModule({
+  imports: [
+    RouterModule.forChild([
+      {
+        path: null,
+        component: PageLayoutComponent,
+        data: {
+          pageLabel: '/configure-overview/cpq',
```

Comment:
I believe we don't need this hardcoded cms `pageLabel`. (Note: we didn't need it for other configurator routes)

Reply:
Picked

Reply:
Done